src/constants/volumePTshader.js

```diff
@@ -862,11 +862,11 @@ int GetNearestLight(Ray R, out vec3 oLightColor, out vec3 Pl, out float oPdf)
 
 // Wi goes toward light sample and is not necessarily perfect reflection of Wo
 // ^Wi   ^N    ^Wo
-//  \    |    /
-//   \   |   /
-//    \  |  /
-//     \ | /
-//      \|/ Pe = volume sample where scattering occurs
+//  \\    |    //
+//   \\   |   //
+//    \\  |  //
+//     \\ | //
+//      \\|// Pe = volume sample where scattering occurs
```

Comment:
was this an auto-formatter change?

Reply:
This is a terrible concession to eslint's no-useless-escaping .  It thought my ascii art was trying to escape nothing, and it's in comments that are embedded in a very long string of GLSL code.  It's hand edited ascii art, not autoformatter.